MoltenVK/MoltenVK/GPUObjects/MVKPipeline.h

```diff
@@ -205,7 +202,6 @@ class MVKPipeline : public MVKVulkanAPIDeviceObject {
 	MVKShaderImplicitRezBinding _indirectParamsIndex;
 	MVKShaderResourceBinding _pushConstantsMTLResourceIndexes;
 	uint32_t _descriptorSetCount;
-	bool _fullImageViewSwizzle;
```

Comment:
I'm worried about this state getting out of sync now that we always go to the configuration, such that we can have a pipeline that was compiled with it on, but bound with it off, so we don't update the swizzle buffer. It's why I put this here in the first place IIRC.

Reply:
Ah...shoot. You're right. And that's actually documented in the notes for the config setting.

Looks like I overreached. I'll close this PR and resubmit one with the smaller changes that just fix the crash.